translations/ja.json

```
@@ -751,7 +751,7 @@
 	"Unable to load profile: {{.error}}": "プロファイルを読み込めません: {{.error}}",
 	"Unable to parse \"{{.kubernetes_version}}\": {{.error}}": "「{{.kubernetes_version}}」を解析できません: {{.error}}",
 	"Unable to parse memory '{{.memory}}': {{.error}}": "メモリー '{{.memory}}' を解析できません: {{.error}}",
-	"Unable to parse version.json: {{.error}}, json: {{.json}}": "",
+	"Unable to parse version.json: {{.error}}, json: {{.json}}": "version.jsonを解析できません: {{.error}}, json: {{.json}}",
```

```suggestion
	"Unable to parse version.json: {{.error}}, json: {{.json}}": "version.json を解析できません: {{.error}}, json: {{.json}}",
```
We need extra space?

---

done